### DISCUSSION TITLE
about pseudo-elements `content:"''"`

### DISCUSSION THREAD

**Discussion by zoy-l** [emotion-js/emotion#2810] [Q&A] [answered]
Hi, just wanted to ask what resistance did encounter with adding quotes to the content value internally, why can't implement `content = JSON.stringify(content); content = content.replace(/\\\\/g, '\\')` internally，did i overlook something 🤔
I verified it myself, there seems to be no problem, maybe I can submit a pr 😆

**Comment by Andarist** [comment]
Could you describe your problem first though? What kind of input doesn't work for you and what would you expect from it?

**Comment by zoy-l** [comment]
maybe i didn't describe clearly, i mean 👇
`  ':after': {
      content: "'hello'" 
    }
`
expect :
`  ':after': {
      content: "hello"  
    }
`
the page displays `hello` normally

**Comment by srmagura** [comment, marked as answer]
The single quotes are required to match plain CSS. In plain CSS, you have to write:

```css
content: 'hello';
```

**Comment by Andarist** [comment]
We'd have to introduce special case for the `content` property in order to handle this. On top of that, we'd have to keep an allowlist of keywords that we shouldn't ever wrap in a string, this, in turn, would be risking this list going in sync with the CSS spec and potentially even us wrapping some newly introduced keywords in quotes. 

Overall I think that we are guided by the "prefer explicit over cute" principle here and this won't be changed.

**Reply by zoy-l** [reply]
Understood, thanks 🤗
